login.css

```diff
@@ -171,8 +188,15 @@ p{
   	top: 240px;
 }
 
+
 #update{
 	color:black;
 	font-size:14px;
 	text-decoration:none;
+	font-family: 'Roboto', sans-serif;
+	font-weight:300;
+}
+
+#update:hover{
+	color:#0C59CF;
```

Comment:
13/08/2021: I apply the hover techniques so that when the user is hover on the forgot password link it display the color navy blue and it can use as an indicator to show user that this is work and it wasn't just text for aesthetic.